bundles/AdminBundle/GDPR/DataProvider/Exporter.php

```diff
@@ -145,7 +145,7 @@ public static function doExportObject(Concrete $object, &$result = [])
 
             if ($fd instanceof Data\Fieldcollections) {
```

Comment:
Fieldcollections can also be null
```suggestion
            if ($fd instanceof Data\Fieldcollections && $value instanceof Fieldcollection) {
```